docs/blog/whats-new-in-phab-3-8.md

```diff
@@ -0,0 +1,265 @@
+# What's new in phabalicious 3.8?
+
+## What has changed?
+
+### PHP 7.3 required
+
+Phabalicious requires now PHP version 7.3. It should work with newer versions, including 8.1
+
+### Better integration with 1Password cli/ connect
+
+* Support for version 2 of the cli
+* added new script callback `get_file_from_1password` which will retrieve an encrypted file from 1password,
+  decrypt it and store it somewhere, suitable when scaffolding.
+
+### Encryption and decryption
+
+Phab supports now encryption and decryption of files and strings. It uses `defuse/php-encryption` under the hood. You
+can encrypt files in a script with
+
+```yaml
+secrets:
+  name-of-secret:
+    question: What is the password?
+
+scripts:
+  encrypt:
+    - encrypt_files(path/to/files/or/folders/to/encrypt/*.ext, path/to/folder/to/store/encrypted/files, name-of-secret)
+  decrypt:
+    - decrypt_files(path/to/files/or/folders/to/decrypt/*.enc, path/to/folder/to/store/decrypted/files, name-of-secret)
+```
+
+The scaffolder has a new callback called `decrypt_assets` which works the same as `copy_assets` but with an additional
+decryption step
+
+```yaml
+scaffold:
+  - decrypt_assets(targetFolder, dataKey, secretName, twigExtension)
+```
+
+You can also use `decrypt()` and `encrypt()` in your scaffold templates using the provided twig-functions:
+
+test.yml
+
+```twig
+data:
+  my_secret_data: "{{ decrypt(my_secret_data, 'name-of-secret') }}"
+  encrypted: "{{ encrypt("Hello world", 'name-of-secret') }}"
+```
+
+### New command `install:from-sql-file`
+
+The command `install:from` and the new command `install:from-sql-file` get reworked to prevent double-work and to
+streamline the process. You can now install the app from an existing sql-file, which translates into an optimized
+flow of
+
+```shell
+phab -cconfig install
+phab -cconfig restore:sql-from-file sql.gz
+phab -cconfig reset
+```
+
+Instead run
+
+```shell
+phab -cconfig install:from-sql-file sql.gz
+```
+
+### Refactoring of data-handling
+
+Phab now handles data differently, by storing from which file or url a piece of data was loaded. This enables phab to
+use relative includes, even for remote configurations. Now you can do things like
+
+base.yml
+
+```yaml
+hosts:
+  test:
+    foo: Hello world
+    bar: Hello from base.yml
+
+```
+
+fabfile.yml:
+
+```yaml
+inheritsFrom:
+  - ./base.yml
+
+needs:
+  - local
+
+hosts:
+  test:
+    foo: Hello mars
+```
+
+This enhances also the introspection possibilities:
+
+### The about command got a `-v` flag
+
+The about-command got enhanced in such a way, that it will show the source of  inheritance when the `-v`-parameter is
+provided. This helps investigating from where a particular value was inherited and in which file it stored. This works
+for local but also for remote sources. `about` will also show all default values applied to a specific configuration.
+
+In our case the ouput of `phab -ctest about` will show the inheritance very in the second column:
+
```

Comment:
The text makes is sound more complicate than it is. 
```
The `about` command has another trick up its sleeve now.

When inheriting configurations from multiple files, it can now display the exact location from which a particular parameter is being picked. 

Thus, finding out if a value is coming from an inherited file from a remote server or some sensible defaults is at your fingertip.

Let's have a look at an example to see this in action. The command displays not only the active value for a particular key but also the location from which it is picked up.

```